ext/drsyscall/drsyscall_record.h

```diff
@@ -90,7 +90,7 @@ typedef enum {
  * value.
  */
 START_PACKED_STRUCTURE
-typedef struct syscall_record_t_ {
+typedef struct ALIGN_VAR(sizeof(reg_t)) syscall_record_t_ {
```

Comment:
Normally a packed structure is read from a file/network, not from memory: so alignment doesn't matter. Would it be better to do that here?